rust-src/concordium_base/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Add a new module `cis3_types` that defines the interface types for CIS3
```

Comment:
This should also mention that `SigningKey` is now exported from ed25519-dalek

Reply:
Yes, I forgot! thanks!